### PR TITLE
Adding PoC for Kruize-VPA Integration

### PR DESCRIPTION
**Pull request by shekhar316**
## Description
This PR adds PoC code for Kruize-VPA Integration. 

### Type of change

- [ ] Bug fix
- [X] New feature
- [ ] Docs update
- [ ] Breaking change (What changes might users need to make in their application due to this PR?)
- [ ] Requires DB changes

## How has this been tested?
Manually on kruize-lm cluster. 

**Test Configuration**
* Kubernetes clusters tested on: kruize-lm

## Checklist :dart:

- [x] Followed coding guidelines
- [x] Comments added
- [ ] Dependent changes merged
- [ ] Documentation updated
- [ ] Tests added or updated

## Additional information
Try out the changes with: quay.io/rh-ee-shesaxen/kruize-vpa-patcher:poc

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+FROM python:3.6.8
+
+WORKDIR /root/src/clever
```
Comment (dinogun) [inline Dockerfile]: Change this to /usr/src/kruize

File: README.md [hunk 2]
```diff
@@ -1,2 +1,90 @@
-# kruize-vpa
-Kruize VPA Integration
+# Kruize - VPA
+
+Kruize VPA Recommender for Kubernetes/ Openshift
```
Comment (dinogun) [inline README.md:3]: Give a reference to clever saying that this code is based off of the clever example

File: Dockerfile (hunk 1 above)
Comment (shekhar316) [reply Dockerfile]: Updated the path. Missed to update it earlier.

File: README.md (hunk 2 above)
Comment (shekhar316) [reply README.md:3]: Updated the README.md file

**Review by dinogun — APPROVED** [review]
LGTM

**Pull request merged**